packages/swayze/lib/src/widgets/default_actions/selection_actions.dart

```diff
@@ -493,6 +493,90 @@ class HeaderSelectionUpdateAction
   }
 }
 
+// TODO: [victor] probably we want to move this to another file.
+class HeaderDragStartAction extends DefaultSwayzeAction<HeaderDragStartIntent> {
```

Comment:
Currently, it is in the `selection_actions` file, we can move it to a better place.